drf_kit/exceptions.py

```diff
@@ -81,3 +82,18 @@ class UpdatingSoftDeletedException(Exception):
     def __init__(self):
         message = "It's not possible to save changes to a soft deleted model. Undelete it first."
         super().__init__(message)
+
+
+def custom_exception_handler(exc, context):
+    resp = views.exception_handler(exc, context)
+    if resp is None:
+        if isinstance(exc, ValidationError):
+            msg = getattr(exc, "message", exc.args[0])
+            if isinstance(msg, (dict, str)):
+                data = msg
+            else:
+                data = exc.messages
+
+            status_code = getattr(exc, "code", "") or status.HTTP_400_BAD_REQUEST
+            return response.Response(data={"errors": data}, status=status_code, exception=exc)
+    return resp
```

Comment:
Use an early return:
```suggestion
    if resp or not isinstance(exc, ValidationError):
        return resp

    msg = getattr(exc, "message", exc.args[0])
    if isinstance(msg, (dict, str)):
        data = msg
    else:
        data = exc.messages

    status_code = getattr(exc, "code", "") or status.HTTP_400_BAD_REQUEST
    return response.Response(data={"errors": data}, status=status_code, exception=exc)
```

In either case, if `resp` is null, and the exception is not `ValidationError`, the return value is going to be `None`. Is that really expected?

If yes, I'd use this for the sake of clarity:

```python
    if resp:
        return resp
    if not isinstance(exc, ValidationError):
        return None
    ...
````